app/src/components/trace/SpanItem.tsx

```diff
@@ -14,7 +14,7 @@ interface SpanItemProps {
   name: string;
   spanKind: string;
   latencyMs: number | null;
-  statusCode: SpanStatusCode;
+  propagatedStatusCode: SpanStatusCode;
```

Comment:
This component is meant to be generic, de-coupled from the API - so it's best to leave these as is and to make the data conform to the contract rather than propagating the change throughout.

Reply:
addressed in [c4f88f4](https://github.com/Arize-ai/phoenix/pull/1824/commits/c4f88f45f6b6b15167a7284e8484e93340e4b1ac)